ansible/roles/common/tasks/main.yml

```diff
@@ -0,0 +1,33 @@
+---
+- name: Install python, if necessary
+  become: true
+  become_method: sudo
+  raw: "test -e /usr/bin/python || dnf install -y {{ common.python_pkg }}"
```

Comment:
I'd probably just include python into `common.packages`

Reply:
Without python installed it seems that ansible's dnf module will fail to run on target machine. I could move all common.packages to be installed in a single task using raw module, but I personally prefer to avoid using raw/command module whenever another option is available. We could also make as pre-requisite that machine have python installed, that would help on the other comment you made below (about $HOME variable)

Reply:
DNF (and Yum) is written in Python, so invoking it pre-determines having Python installed and available, but let me try to make sure that there are no Python2/3 problems.

+1 on avoiding raw/command/shell as it sort of defeats the purpose of Ansible :)

Reply:
Yes, you are right. I am using an lxc container with fedora 26 image for testing. Vanilla image has dnf and python3 installed, ansible complains about missing python binary.